### ISSUE TITLE
Is the address-family for BGP a list ?

### ISSUE THREAD

**Issue by robric** [boucadair/attachment-circuit-model#8]
As per title, just to check that we can have several AF with the current model (not clear for me - I see leaf)

**Comment by boucadair** [comment]
That's correct. This leaf can take one of these values: 'ipv4', 'ipv6', or 'dual-stack'.
